MapboxSearch/sdk/src/test/java/com/mapbox/search/location/LocationEngineAdapterTest.kt

```diff
@@ -1,174 +0,0 @@
-package com.mapbox.search.location
-
-import android.app.Application
-import android.location.Location
-import com.mapbox.android.core.location.LocationEngine
-import com.mapbox.android.core.location.LocationEngineCallback
-import com.mapbox.android.core.location.LocationEngineResult
-import com.mapbox.android.core.permissions.PermissionsManager
-import com.mapbox.geojson.Point
-import com.mapbox.search.base.location.LocationEngineAdapter
-import com.mapbox.search.base.logger.reinitializeLogImpl
-import com.mapbox.search.base.logger.resetLogImpl
-import com.mapbox.search.base.utils.TimeProvider
-import com.mapbox.test.dsl.TestCase
-import io.mockk.every
-import io.mockk.mockk
-import io.mockk.mockkStatic
-import io.mockk.slot
-import io.mockk.unmockkStatic
-import org.junit.jupiter.api.AfterAll
-import org.junit.jupiter.api.AfterEach
-import org.junit.jupiter.api.BeforeAll
-import org.junit.jupiter.api.BeforeEach
-import org.junit.jupiter.api.TestFactory
-
-internal class LocationEngineAdapterTest {
```

Comment:
This is a duplicating test left since the `LocationEngieAdapter` moved to the `base` module.